setup.py

```diff
@@ -57,7 +57,7 @@ def run(self):
 
 
 setup(name='citrine',
-      version='0.62.3',
+      version='0.63.3',
```

Comment:
Strange version bump

Reply:
Oh I see what you mean, yeah I'll fix that